### PR TITLE
Add karto configs

### PR DESCRIPTION
**Pull request by loicgasser**

### CODE REVIEW

File: configs/dkm10_bodenbedeckung.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "layerBodId": "dkm10_bodenbedeckung",
```
Comment (stebie) [inline configs/dkm10_bodenbedeckung.json]: Is the ordering of these fields defined somehow?

File: vectorforge/models/stopo.py [hunk 2]
```diff
@@ -7,12 +7,26 @@
...
+if bases.get('stopo_master') is None:
     init()
-Base = bases.get('stopo_int')
-
+Base = bases.get('stopo_master')
+
+GeomMultiPoint = Geometry(
+        geometry_type='MULTIPOINT',
+        dimension=2,
+        srid=21781)
```
Comment (stebie) [inline vectorforge/models/stopo.py]: What's 'srid' ?

**Comment by stebie** [comment]
Thanks for adding the skeletons of the new carto configs !

**Comment by loicgasser** [comment]
You re welcome. Srid stands for spatial reference id.

**Pull request merged**